avaframe/ana3AIMEC/dfa2Aimec.py

```diff
@@ -112,58 +108,115 @@ def dfaBench2Aimec(avaDir, cfg, simNameRef, simNameComp):
 
         Returns
         --------
+        inputsDF: dataFrame
+            simulation dataframe with optionally path to mass file
         pathDict: dict
             dictionary with paths to simulation results
 
     """
 
     cfgSetup = cfg['AIMECSETUP']
-    comModules = cfg['AIMECSETUP']['comModules'].split('|')
+    comModules = cfgSetup['comModules'].split('|')
     # get directories where simulation results can be found for both modules
     inputDirRef, inputDirComp, pathDict = getCompDirs(avaDir, cfgSetup)
 
     # Load all infos on reference simulations
     refData, resTypeRefList = fU.makeSimFromResDF(avaDir, None, inputDir=inputDirRef, simName=simNameRef)
+    if len(refData.index) == 0:
+        message = ('Found no simulation matching the reference criterion, '
+                   'there should be one')
+        log.error(message)
+        raise ValueError(message)
+    elif len(refData.index) > 1:
+        message = ('Found multiple simulations matching the reference criterion,'
+                   'there should be only one reference')
+        log.error(message)
+        raise ValueError(message)
+    else:
+        colorParameter = False
```

Comment:
why is this set to false?

Reply:
can the sims from the comparison module become color coded?

Reply:
This is for color coding, it is set to false because for the aimec comparison I do not have the config info...
In the future (see issue about aimec that I opened) we will try to load the config for the ref and comp simulations and if we succeeded, we will set  colorParameter to True and be able to color code the results